lib/auth/touchid/api.go

```diff
@@ -42,14 +44,29 @@ import (
 var (
 	ErrCredentialNotFound = errors.New("credential not found")
 	ErrNotAvailable       = errors.New("touch ID not available")
+
+	// PromptPlatformMessage is the message shown before Touch ID prompts.
+	PromptPlatformMessage = "Using platform authenticator, follow the OS prompt"
+	// PromptWriter is the writer used for prompt messages.
+	PromptWriter io.Writer = os.Stderr
 )
 
+func promptPlatform() {
+	if PromptPlatformMessage != "" {
```

Comment:
Why do we need this check?

Reply:
It's tied to the question above: if for some reason we want to disable the message, an outside package can set it to empty and the Println is skipped.